src/Bindings/ManualBindings.cpp

```diff
@@ -1493,6 +1482,18 @@ static int tolua_cPluginManager_GetCurrentPlugin(lua_State * S)
 
 
 
+static int tolua_cPluginManager_GetPlugin(lua_State * tolua_S)
```

Comment:
A little unrelated, but maybe a `cPluginManager::PluginExists(a_PluginName)` could be usefull. Previously `cPluginManager::GetPlugin` could be used for that.